core/templates/filters/translate.pipe.ts

```diff
@@ -89,6 +89,10 @@ export class TranslatePipe implements PipeTransform, OnDestroy {
   }
 
   /**
+   * Every pipe is supposed to have a function named transform as Pipes
```

Comment:
I don't understand what you mean by "as Pipes PipeTranform". Could you explain that part in standard prose English?

Also, you have misspelled "transform" in lots of places in this comment.

Reply:
changed now